src/components/Playbar/index.tsx

```diff
@@ -137,7 +137,7 @@ export function PlaybarUI(props: PlaybarUIProps): ReactElement {
                         <Swap className="w-3" />
                     </Button>
                 )}
-                {!hideEventDetails && (
+                {(!hideEventDetails || showSwap) && (
```

Comment:
in the playbar - eventDetails should be shown if the swap icon is present - which means we're playing the audio of another event